test/loopback.test.js

```diff
@@ -631,4 +631,73 @@ describe('loopback', function() {
       });
     });
   });
+
+  describe('new remote method configuration', function() {
+    it('Remote method - isStatic specified in method name', function() {
+      var TestModel = loopback.createModel(uniqueModelName);
+      loopback.configureModel(TestModel, {
+        dataSource: null,
+        methods: {
+          staticMethod: {
+            http: { path: '/static' }
+          },
+          'prototype.instanceMethod': {
+            http: { path: '/instance' }
+          }
+        }
+      });
+
+      var methodNames = TestModel.sharedClass.methods().map(function(m) {
+        return m.stringName.replace(/^[^.]+\./, ''); // drop the class name
+      });
```

Comment:
Could you please extract these three lines of code and reuse in all places where it's duplicated now?

Example usage:

```js
var methodNames = getAllMethodNamesWithoutClassName(TestModel);
```